build.gradle

```diff
@@ -1,27 +1,32 @@
 buildscript {
+    ext{
+        kotlin_version = '1.7.10'
+        coroutines_version = '1.6.3'
+    }
+
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
+
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.3'
-        classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
+        classpath 'com.android.tools.build:gradle:7.2.2'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
 ext {
     minSdkVersion = 9
-    targetSdkVersion = 27
-    compileSdkVersion = 27
+    targetSdkVersion = 29
```

Comment:
I wonder why we are not targeting 33?

Reply:
cause that's since when i've been wanting to update this app 😂. ✅